lombok.config

```diff
@@ -0,0 +1 @@
+lombok.anyConstructor.addConstructorProperties=true
```

Comment:
Good catch. I read https://github.com/rzwitserloot/lombok/issues/1563 and seems like this is the only solution for now. However, since this only affects tests, what do you think about moving this file to the test source folder?